3rdParty/lz4/build/cmake/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ else()
     LANGUAGES C)
 endif()
 
-cmake_minimum_required (VERSION 2.8.6)
+cmake_minimum_required (VERSION 3.21)
```

Comment:
Is there a specific reason for this change? It is a minimal version, so if you did not change any other stuff inside this directory that requires a larger version, I don't see why you need to change this number.

Reply:
@jvolmer @KVS85 the problem is that newer CMake generate some scary deprecation warnings. If you try to compile yourself this makes it very hard to spot the real errors.

Reply:
I assume, if we upgrade LZ4 it will have a newer version, because 2.8.6 is no longer support by the latest CMake.

Reply:
Please note that this is not about upgrading any 3rdParty libs. I will not touch them in this PR. We can do this seperately.

Reply:
ok, that's fine for me